lib/Mage/System/Ftp.php

```diff
@@ -359,7 +359,7 @@ public function get($localFile, $remoteFile, $fileMode = FTP_BINARY, $resumeOffs
      * ftp_nlist wrapper
      *
      * @param string $dir
-     * @return bool
+     * @return array
```

Comment:
https://www.php.net/manual/en/function.ftp-nlist.php
> Returns an array of filenames from the specified directory on success or false on error. 
```suggestion
     * @return string[]|false
```